src/verifications/verifications.controller.ts

```
@@ -0,0 +1,32 @@
+  @ApiOperation({ summary: 'Verify Application VCS' })
+  @ApiBody({
+    type: VerifyApplicationVcsRequestDto,
+    description: 'The application ID to verify',
+    examples: {
+      default: {
+        value: { applicationId: "1" }, // Correctly define the payload structure
+      },
+    },
+  })
+  async verifyApplicationVcs(
+    @Body() verifyApplicationVcsRequestDto: VerifyApplicationVcsRequestDto
```

ESLint

[error] Insert `,`

(prettier/prettier)